src/main/java/sg/ncl/MainController.java

```diff
@@ -2283,11 +2301,24 @@ private void setSessionVariables(HttpSession session, String loginEmail, String
         session.setAttribute("sessionLoggedEmail", loginEmail);
         session.setAttribute("id", id);
         session.setAttribute("name", user.getFirstName());
+        // FIXME: get user type from token
+//        session.setAttribute("userType", "normal");
+        session.setAttribute("userType", "admin");
     }
 
     private void removeSessionVariables(HttpSession session) {
         session.removeAttribute("sessionLoggedEmail");
         session.removeAttribute("id");
         session.removeAttribute("name");
+        // FIXME: get user type from token
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1134)